ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -326,13 +327,21 @@ export const ListSearch = fp.flow(withCdrVersions(), withCurrentWorkspace(), wit
       }
     }
 
+    getCriteriaInfoByDomain(domain, value, surveyName) {
+      const {workspace: {id, namespace}, searchContext: {standard}} = this.props;
+      // Remove the standard null check once https://precisionmedicineinitiative.atlassian.net/browse/RW-6982 is done
+      return serverConfigStore.get().config.enableStandardSourceDomains
+          ? cohortBuilderApi().findCriteriaByDomain(namespace, id, domain, standard == null ? true : standard, value, surveyName)
```

Comment:
Same comment here, we should be able to remove the condition and just use `standard`.

Reply:
The condition is required because i was getting null from concept set context.

Reply:
Ah, ok. I didn't test concept search the first time through.